### DISCUSSION TITLE
JavaScript heap out of memory

### DISCUSSION THREAD

**Discussion by mwaaas** [coder/code-server#5260] [Q&A] [answered]
Getting the following error on coder 

Committing semi space failed. Allocation failed - JavaScript heap out of memory

my instance memory looks good

**Comment by jsjoeio** [comment, marked as answer]
Can you provide more information or logs or a screenshot?

**Reply by mwaaas** [reply]
Hey Jsjoeio, 

The error disappeared after starting the server again. Next time it happens will make sure to take screenshot.
